1.8.9/src/main/java/io/github/axolotlclient/modules/zoom/Zoom.java

```diff
@@ -87,6 +95,7 @@ private static boolean shouldStart() {
 
 	private static void start() {
 		active = true;
+		Keyboard.enableRepeatEvents(true);
```

Comment:
> This might cause quite weird behaviour as it applies to all keys and can even be used to click automatically iirc
well, maybe it isn't a great solution then..

> You don't need to change this - Just saying so you're aware
it could be left out if we use `keybind.isPressed()` below but that made it increase too much (which made it look weird)